source/docs/prismic/expose-content.md

```diff
@@ -0,0 +1,368 @@
+---
+id: expose-content
+title: Expose Prismic Content
+---
+
+The Prismic Front-Commerce module provides a loader and the infrastructure to expose Prismic based Content in Front-Commerce's GraphQL API.
+
+## Prerequisites
+
+To benefit from this API, you first need to [install the Prismic module](/docs/prismic/installation.html). You will also need [a custom GraphQL module](/docs/essentials/extend-the-graphql-schema.html) to define a GraphQL schema matching the content you want to expose and implement the corresponding resolver. It also recommended to have read [the Prismic Core Concepts documentation](https://prismic.io/docs/core-concepts).
+
+## Prismic loader API
+
+### Methods to query content
+
+The Prismic loader has the following API to request Content from Prismic:
+
+* `loadSingle(typeIdentifier[, options])` returns [an object of type `Content`](https://gitlab.com/front-commerce/front-commerce-prismic/-/blob/0.5.0/prismic/server/modules/prismic/core/loaders/Content.js) or throws an error if no such Content exists.
```

Comment:
I'm a bit afraid by the amount of links to the code we added lately, and how difficult it might be to maintain.

Since we don't yet have any process for ensuring their correctness it'll either:
- lead to dead links at some point if using a branch name for the reference (e.g: `main`)
- or lead to outdated documentation/pointer if using a commit hash or tag

I tend to prefer the `main` branch link, so that a dead link would hopefully be noticed and trigger a documentation review (as the documentation will likely be outdated if the file doesn't exist anymore)

Reply:
funny, I had the same questionning and end up with the opposite conclusion ;-) but without a strong opinion, so I'll switch to `main` instead

Reply:
fixed in d129005